app-config.yaml

```
@@ -149,6 +149,13 @@ catalog:
         - allow: [Template]
 argocd:
   baseUrl: https://localhost:8080
+  username: ${ARGOCD_USERNAME}
+  password: ${ARGOCD_PASSWORD}
+  appLocatorMethods:
```

If you modify this to be an empty array, it should revert back to the old functionality of using the proxy for ArgoCD instead and the old e2e tests should succeed.
```suggestion
  appLocatorMethods: []
```

Maybe add the lines below in as a comment instead.

---

trying that now.

---

we don't use cypress internally at this moment however a PR is coming to enable it after our teams went through training. I am running that locally now as well for testing.Thanks for the suggestion.

---

Yup, I think this change and the config boolean check on the frontend side should revert to the original proxy approach for this plugin --> E2E tests should pass like before.